packages/common-ui-web/test/components/TakePictureButton.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TakePictureButton } from '../../src';
+
+const TAKE_PICTURE_BTN_TEST_ID = 'take-picture-btn';
+const TAKE_PICTURE_OUTER_BTN_TEST_ID = 'take-picture-outer-btn';
```

Comment:
rename this variable to : `const OUTER_LAYER_TEST_ID = 'take-picture-btn-outer-layer';`